test/unit/conftest.py

```diff
@@ -152,3 +152,9 @@ def apiver(request):
 def apiver_int(apiver):
     """Get apiver as an int, e.g. `2`."""
     return int(apiver[1:])
+
+
+@pytest.fixture(scope='session')
+def apiver_deps():
+    import apiver_deps  # noqa: F401
+    return apiver_deps
```

Comment:
Too clever for me, need help

Reply:
this is more of preparation for this: https://github.com/reef-technologies/b2-sdk-python/blob/master/noxfile.py#L110

Reply:
Hm, I read that comment as "we want to have different nox sessions for each apiver test". Smth like:
```python
@nox.parametrize('apiver', [0, 1, 2, 3])
```
https://nox.thea.codes/en/stable/config.html#parametrizing-sessions

Ok, whatever, if you introduce the fixture then maybe replace everything with a fixture? It would be weird if one test uses the fixture and others don't